lib/puppet/file_serving/fileset.rb

```diff
@@ -105,7 +106,7 @@ def initialize_from_hash(options)
       begin
         send(method, value)
       rescue NoMethodError
-        raise ArgumentError, _("Invalid option '%{option}'") % { option: option }, $!.backtrace
+        raise ArgumentError, _("Invalid option '%{option}'") % { option: option }, $ERROR_INFO.backtrace
```

Comment:
I think it's preferable to use

```ruby
rescue Puppet::ExecutionFailure => e
  raise ArgumentError, _("Invalid option '%{option}'") % { option: option }, e.backtrace
```
and then drop the `require 'English'` above.

Reply:
Making sure I follow, then should Style/SpecialGlobalVars  not be enabled since that cop is what made those changes?